lib/cocina/models/mapping/to_mods/event.rb

```diff
@@ -80,12 +80,22 @@ def write_event(cocina_event_type, dates, locations, names, notes, attributes, i
                 xml.publisher name.value, name_attributes(name) if name
               end
               Array(notes).each do |note|
-                write_note(note)
+                if note.parallelValue.present?
+                  write_parallel_note(note)
```

Comment:
Wondering why this is showing up as untested?

Reply:
We don't have a test for event.rb here at all... trying to simplify this logic just a bit to avoid the coverage issue.